### PR TITLE
Add _builtin suffix to builtin names.

### PR DESCRIPTION
**Pull request by alonh5**
# TITLE

## Description

Description of the pull request changes and motivation.

## Checklist
- [ ] Linked to Github Issue
- [ ] Unit tests added
- [ ] Integration tests added.
- [ ] This change requires new documentation.
  - [ ] Documentation has been added/updated.
  - [x] CHANGELOG has been updated.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 #### Upcoming Changes
 
+* Add _builtin suffix to builtin names e.g.: output -> output_builtin [#1005](https://github.com/lambdaclass/cairo-rs/pull/1005)
```
Comment (Oppen) [inline CHANGELOG.md:5]: It should mention it's (technically) a breaking change. Any code __not__ expecting the suffix would break.

**Review by Oppen — CHANGES_REQUESTED** [review]
Minor change to the CHANGELOG needed.
Also, I know it's not documented or enforced anywhere (yet), but we're planning on using [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/) so updating the CHANGELOG could be automated or at least assisted by the CI, so I would appreciate if we could start following those.

File: CHANGELOG.md (hunk 1 above)
Comment (pefontana) [reply CHANGELOG.md]: Done!
[4307125](https://github.com/lambdaclass/cairo-rs/pull/1005/commits/43071257e313440c1d272890badbbfaf08b439d8)

**Comment by pefontana** [comment]
> Minor change to the CHANGELOG needed. Also, I know it's not documented or enforced anywhere (yet), but we're planning on using [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/) so updating the CHANGELOG could be automated or at least assisted by the CI, so I would appreciate if we could start following those.

Thanks for the remainder @Oppen !
Didn't get used to it yet

**Comment by Oppen** [comment]
> Thanks for the remainder @Oppen !
> Didn't get used to it yet

No worries. Do note it's not a blocker. Just if you need to change something else, seize the opportunity. Until documented/enforced by CI I won't consider such thing a must because it would be unfair to reject PRs based on something we didn't tell was needed :)

**Review by Oppen — APPROVED** [review]

**Review by fmoletta — APPROVED** [review]

**Pull request merged**